tests/test_cli.py

```diff
@@ -94,3 +94,21 @@ def test_compile_command(tmp_path):
     result = runner.invoke(console_main, ["compile", "--plugin-dir", tmp_path])
     assert result.exception is None
     assert result.output == ""
+
+
+def test_command_update(tmp_path):
+    runner = CliRunner()
+    result = runner.invoke(console_main, ["update", "--plugin-dir", tmp_path])
+    assert (
+        f"Was not possible to find 'src/hook_specs.h' file in {tmp_path}"
+        == result.exception.args[0]
+    )
+
+    plugin_src_folder = tmp_path / "plugin_a" / "src"
+    plugin_src_folder.mkdir(parents=True)
+    (plugin_src_folder / "hook_specs.h").write_text(data="", encoding="utf-8")
+    result = runner.invoke(
+        console_main, ["update", "--plugin-dir", plugin_src_folder.parent]
+    )
+    assert result.exception is None
+    assert result.output == ""
```

Comment:
You could check that `(plugin_src_folder / "hook_specs.h")` is no longer empty.

Reply:
It is already tested in another place when the function that generates the file is implemented.
Here, only the CLI commands are tested. See the other CLI commands Tests.

Reply:
Although, I will check as simple as possible, like `size > 0`

- [ ] check if file is not empty